packages/api/src/apps/api/app.ts

```diff
@@ -17,6 +17,7 @@ export default async (env: ProcessEnv) => {
   assert(env.EXPRESS_PORT, "requires EXPRESS_PORT");
   assert(env.zrxBaseUrl, "requires zrxBaseUrl");
   assert(env.MULTI_CALL_ADDRESS, "requires MULTI_CALL_ADDRESS");
+  assert(env.MULTI_CALL_2_ADDRESS, "requires MULTI_CALL_2_ADDRESS");
```

Comment:
should we just require a multicall 2 address and use it for both multicall1 and 2? Also i dont remember if multicall 2 is just a drop in replacement for 1, maybe we should just replace the `multicall` reference with the `multicall2` implementation?

Reply:
we can definitely do that